src/sass/app_2/_common/base/constants.scss

```diff
@@ -33,6 +34,7 @@ $COLOR_DARK_BLUE_6         : #555975;
 $COLOR_DARK_BLUE_7         : #7f8397;
 $COLOR_DARK_BLUE_8         : rgba(127, 131, 151, 0.3);
 $COLOR_DARK_GREEN_GRADIENT : linear-gradient(to top, rgba(16, 19, 32, 0), rgba(0, 148, 117, 0.16));
+$COLOR_DARK_TRANSPARENT    : rgba(31, 36, 61, 0.8);
```

Comment:
@cakasuma I don't see these colours in Zeplin style guide.

Reply:
yes, i was not aware of the `transparentize mixin`. i'll fix it